lib/routes/routeBackbeat.js

```diff
@@ -305,9 +309,36 @@ POST /_/backbeat/multiplebackenddata/<bucket name>/<object key>
     ?operation=completempu
 POST /_/backbeat/multiplebackenddata/<bucket name>/<object key>
     ?operation=puttagging
+GET /_/backbeat/multiplebackenddata/<bucket name>/<object key>
 POST /_/backbeat/batchdelete
 */
 
+function headObject(request, response, log, callback) {
+    let locations;
+    try {
+        locations = JSON.parse(request.headers['x-scal-locations']);
```

Comment:
That should work, as long as we make sure we don't pass a full array of sproxyd locations here (which could be in MBs while HTTP header size is usually limited to a few KB at most).

Reply:
Okay, I can add a check for that before backbeat sends the request. It appears that sending in the body (as is done with batch-delete) is not supported by the AWS schema when using GET.

Reply:
Just to follow-up on this. I've updated backbeat to only send one array entry as this is sufficient for the data wrapper to perform the head request.